packages/app/package.json

```diff
@@ -13,5 +13,9 @@
   "devDependencies": {
     "@types/react": "^18.2.55",
     "@types/react-native": "0.73.0"
+  },
+  "peerDependencies": {
+    "lodash": "^4.17.21",
+    "zustand": "^3.7.2"
```

Comment:
we aren't listing a lot of the dependencies we import from `packages/app/src/` here – idk best practice but it seems best to be explicit about what's needed (e.g. let's say we add a new package `apps/tlon-vr` and we want to import `@tloncorp/apps` – having `peerDependencies` listed would let us know what we need in that new package)